README.md

```diff
@@ -10,25 +10,10 @@ Kubernetes and Mesos are a match made in heaven.
 Kubernetes enables the Pod (group of co-located containers) abstraction, along with Pod labels for service discovery, load-balancing, and replication control.
```

Comment:
Suggestion:
Kubernetes enables the Pod which an abstraction of a group of co-located containers.  Each Pod uses Labels for service discovery, load-balancing, and replication control.